tests/consensus_tests/test_disable_url_snapshot_recovery.py

```
@@ -0,0 +1,73 @@
+import pathlib
+
+import requests
+
+from .fixtures import create_collection
+from .utils import *
```

Ruff (0.15.10)

[error] `from .utils import *` used; unable to detect undefined names

(F403)